### PR TITLE
AWS Indexing

### PR DESCRIPTION
**Pull request by thefeli73**
currently believed to be working:
Polemos start -> handle config -> index instances (aws only)

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,8 +1,61 @@
...
+			found = true
+		}
+	}
+	u := uuid.New()
+	newService := state.Service{
+		ID: state.CustomUUID(u),
+		CloudID: cloudID,
+		ServiceIP: serviceIP}
+	return newService, found
 }
```
Comment (GreenPenguino) [inline main.go]: Configure VSCode to add newline on save

File: mtdaws/utils.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+		for _, reservation := range page.Reservations {
+			for _, instance := range reservation.Instances {
+				instances = append(instances, &instance)
+			}
+		}
+	}
+
+	return instances, nil
+}
```
Comment (GreenPenguino) [inline mtdaws/utils.go]: Newline

File: state/config.go [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    if err != nil {
+        return err
+	}
+
+	err = ioutil.WriteFile(filename, yamlBytes, 0644)
+	if err != nil {
+        return err
+	}
+    return nil
+}
```
Comment (GreenPenguino) [inline state/config.go]: Newline!!!!

**Review by GreenPenguino — CHANGES_REQUESTED** [review]
Aaaargggg!!!

File: state/config.go (hunk 3 above)
Comment (thefeli73) [reply state/config.go]: fixed in latest commit

File: mtdaws/utils.go (hunk 2 above)
Comment (thefeli73) [reply mtdaws/utils.go]: fixed in latest commit

File: main.go (hunk 1 above)
Comment (thefeli73) [reply main.go]: fixed in latest commit

**Review by GreenPenguino — APPROVED** [review]
Go brrrrrrr!!!

**Pull request merged**